lib/spack/spack/cmd/find.py

```diff
@@ -140,9 +140,12 @@ def find(parser, args):
 
     # Exit early if no package matches the constraint
     if not query_specs and args.constraint:
-        msg = "No package matches the query: {0}"
+        msg = "No installed package matches the query: {0}"
```

Comment:
Same here, if I have no installed packages, `spack find` should say "No installed package".